source/statements.tex

```diff
@@ -921,6 +921,10 @@
 The expression \placeholder{p}\tcode{.return_void()}
 shall be a prvalue of type \keyword{void}.
 \end{itemize}
+\begin{note}
+The overload resolution associated with a \tcode{co_return} statement is affected
```

Comment:
Which overload resolution are we talking about? The one for `p.return_void()`?

Reply:
Oops, good catch: This note was meant to apply to `co_return` as a whole (and "the overload resolution" would be the one associated with `p.return_value(operand)`, because that's the only place "the operand" makes sense), but then it probably makes sense to move it up into the bulleted list, right after we talk about `p.return_value(operand)` and before we introduce `p.return_void()` at all. Done.